src/huggingface_hub/constants.py

```diff
@@ -13,6 +13,7 @@
 TF_WEIGHTS_NAME = "model.ckpt"
 FLAX_WEIGHTS_NAME = "flax_model.msgpack"
 CONFIG_NAME = "config.json"
+MODELCARD_NAME = "README.md"
```

Comment:
You're setting `MODELCARD_NAME` here but using the `repocard` term elsewhere - what should it be? I would favor having all instances be `REPOCARD_NAME` if you're looking for a repo-type-agnostic name